src/main/java/fr/quatrevieux/araknemu/game/fight/turn/action/ActionType.java

```diff
@@ -29,7 +29,8 @@
     NONE(0, 0),
     MOVE(1, 2),
     CAST(300, 0),
-    CLOSE_COMBAT(303, 0);
+    CLOSE_COMBAT(303, 0),
+    CANT_MOVEOUT(104, 0);
```

Comment:
Can't moveout is not an action but a result (like cast failed) : it must not be declared here, but directly returned by `ActionResult#action()`